lib/vsc/filesystem/posix.py

```diff
@@ -330,7 +330,7 @@ def make_symlink(self, target, obj=None, force=True):
     def is_dir(self, obj=None):
         """Check if it is a directory"""
         obj = self._sanity_check(obj)
-        # do symlinks count ?
+        return os.path.isdir(obj)
```

Comment:
so add in the docstring that a symlink to a directory is considered a directory (we don't use the `is_dir` anywhere, so this change is harmless)